app/styles/components/c-cover-page.scss

```diff
@@ -51,6 +51,10 @@
     }
   }
 
+  .stub::after{
+    height: 0px;
+  }
+
```

Comment:
when loading the default "empty" slide, the empty <h1> would still have the red :after element. this removes that just for that stub slide